job/tensorflowtrainingjob/ps/job.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: TensorFlowTrainingJob
+metadata:
+  name: keras-mnist-trainingjob-ps
+spec:
+  tensorboardSpec:
+    trainingLogFilesets:
+      - t9k://pvc/tutorial/tutorial-examples/job/tensorflowtrainingjob/ps/log
+    image: registry.tensorstack.cn/t9k/tensorflow-2.5.0:cpu
+  replicaSpecs:
+    - type: chief
+      replicas: 1
+      restartPolicy: ExitCode
+      template:
+        spec:
+          securityContext:
+            runAsUser: 1000
+          containers:
+            - command:
+                - python3
+                - job/tensorflowtrainingjob/ps/keras_mnist_trainingjob_ps.py
+                - "--log_dir"
+                - "job/tensorflowtrainingjob/ps/log"
+                - "--no_cuda"
+              workingDir: /t9k/mnt/tutorial-examples
+              image: registry.tensorstack.cn/t9k/tensorflow-2.5.0:cpu
```

Comment:
use newer release of tensorflow?

Reply:
change to `registry.tensorstack.cn/t9k/tensorflow-2.11.0:cpu-sdk-0.5.1`